CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+## Contributing
+
+We welcome contributions to this project in many forms, and there's
+always plenty to do! Whether it is to fix a bug, improve the documentation,
+or add support for any new DLT platform.
+
+Please familiarize yourself with the
+[documentation](https://labs.hyperledger.org/weaver-dlt-interoperability/),
+the [Design
+Principles](https://labs.hyperledger.org/weaver-dlt-interoperability/docs/external/design-principles)
+and our
+[RFCs](https://labs.hyperledger.org/weaver-dlt-interoperability/docs/external/specifications).
```

Comment:
The specifications page should contain a back link to the RFCs in github. I'll add that in another PR.